chain/src/txhashset/txhashset.rs

```diff
@@ -1323,28 +1323,39 @@ impl<'a> Extension<'a> {
 
 		let mut kern_count = 0;
 		let total_kernels = pmmr::n_leaves(self.kernel_pmmr.unpruned_size());
+		let mut tx_kernels: Vec<TxKernel> = Vec::with_capacity(1_024);
 		for n in 1..self.kernel_pmmr.unpruned_size() + 1 {
 			if pmmr::is_leaf(n) {
 				let kernel = self
 					.kernel_pmmr
 					.get_data(n)
 					.ok_or::<Error>(ErrorKind::TxKernelNotFound.into())?;
 
-				kernel.verify()?;
+				tx_kernels.push(kernel.kernel);
 				kern_count += 1;
 
-				if kern_count % 20 == 0 {
+				// batch on every 1024 kernels
+				if kern_count & 0x0400 == 0 {
```

Comment:
These `& 0x0400` are _really_ hard to read at a glance. Can we use modulo `%` here for simplicity?
The counts are integers, so it makes sense to just use integer modulo (rather than bitwise and).

Reply:
That's for efficiency, `& 0x0400` is much faster than ` % sth`. For readable, that's why the comments line is there.

Reply:
> That's for efficiency

This is almost certainly not the bottleneck here. I'd much prefer the code be easy to read (and self documenting).

Reply:
😄 Okay, let me change this.